wagtailmenus/conf/defaults.py

```diff
@@ -89,3 +87,5 @@
 # ----------
 # Deprecated
 # ----------
+
+ADD_EDITOR_OVERRIDE_STYLES = True
```

Comment:
These styles don't seem to be needed anymore. The wagtail UI deals with them perfectly.